build.gradle.kts

```diff
@@ -37,6 +37,8 @@ allprojects {
     }
 
     tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
-        kotlinOptions.allWarningsAsErrors = true
```

Comment:
No longer works?

Reply:
or just for consistency?

Reply:
it works on both ways I just prefer this syntax